### ISSUE TITLE
The `community.grafana.grafana_dashboard` module doesn't accept the `url` parameter, only works using `grafana_url`

### ISSUE THREAD

**Issue by itaytalmi** [ansible-collections/community.grafana#250]
##### SUMMARY

Attempting to create a Grafana dashboard using the `community.grafana.grafana_dashboard` fails when the `url` parameter is used. However, it works perfectly when `grafana_url` is specified (although `grafana_url` is a just the alias according to the documentation - https://docs.ansible.com/ansible/latest/collections/community/grafana/grafana_dashboard_module.html). Using other modules, such as `community.grafana.grafana_datasource`, the `url` parameter works fine.
See below steps to reproduce.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

`community.grafana.grafana_dashboard`

##### ANSIBLE VERSION

```paste below
ansible [core 2.12.5]
  config file = None
  configured module search path = ['/home/k8s/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  ansible collection location = /home/k8s/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
```

##### COLLECTION VERSION

```paste below
$ ansible-galaxy collection list community.grafana

# /usr/local/lib/python3.8/dist-packages/ansible_collections
Collection        Version
----------------- -------
community.grafana 1.4.0
```

##### CONFIGURATION

```paste below
None...
```

##### OS / ENVIRONMENT

```paste below
$ cat /etc/os-release

NAME="Ubuntu"
VERSION="20.04.4 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.4 LTS"
```

##### STEPS TO REPRODUCE

Use this sample playbook:

```yaml
- hosts: localhost
  connection: local
  tasks:
    - name: Import Grafana RabbitMQ-Overview Dashboard
      community.grafana.grafana_dashboard:
        url:  https://my-grafana.my-domain.com
        url_username: admin
        url_password: your-p@ssw0rd
        dashboard_id: 10991
        state: present
        folder: General
        overwrite: yes
        validate_certs: no
```

`ansible-playbook playbook.yml` should result in the following error:

```text
TASK [Import Grafana RabbitMQ-Overview Dashboard] *************************************************************************************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: KeyError: 'grafana_url'
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655457843.877443-374914-33418404310105/AnsiballZ_grafana_dashboard.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655457843.877443-374914-33418404310105/AnsiballZ_grafana_dashboard.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655457843.877443-374914-33418404310105/AnsiballZ_grafana_dashboard.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', init_globals=dict(_module_fqn='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', _modlib_path=modlib_path),\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_community.grafana.grafana_dashboard_payload_mpq_xqz_/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py\", line 559, in <module>\n  File \"/tmp/ansible_community.grafana.grafana_dashboard_payload_mpq_xqz_/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py\", line 514, in main\nKeyError: 'grafana_url'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}

PLAY RECAP ****************************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

However, changing the `url` parameter to `grafana_url` works.

Working playbook:

```yaml
- hosts: localhost
  connection: local
  tasks:
    - name: Import Grafana RabbitMQ-Overview Dashboard
      community.grafana.grafana_dashboard:
        grafana_url:  https://my-grafana.my-domain.com
        url_username: admin
        url_password: your-P@ssw0rd
        dashboard_id: 10991
        state: present
        folder: General
        overwrite: yes
        validate_certs: no
```

`ansible-playbook playbook.yml` succeeds.

```text
ok: [localhost]

TASK [Import Grafana RabbitMQ-Overview Dashboard] *************************************************************************************************************************************************************************
changed: [localhost]

PLAY RECAP ****************************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

According to the documentation, the `url` parameter is actually the official one, and `grafana_url` should be an alias.
https://docs.ansible.com/ansible/latest/collections/community/grafana/grafana_dashboard_module.html

*Additional parameters might be affected by this issue.

##### EXPECTED RESULTS

The module should accept the `url` parameter and succeed. However, it seems to accept on the alias.

##### ACTUAL RESULTS

```paste below
$ ansible-playbook playbook.yml -vvvv
ansible-playbook [core 2.12.5]
  config file = None
  configured module search path = ['/home/k8s/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  ansible collection location = /home/k8s/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible-playbook
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
No config file found; using defaults
setting up inventory plugins
host_list declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
Skipping due to inventory source not existing or not being readable by the current user
script declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
auto declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
Skipping due to inventory source not existing or not being readable by the current user
yaml declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
Skipping due to inventory source not existing or not being readable by the current user
ini declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
Skipping due to inventory source not existing or not being readable by the current user
toml declined parsing /etc/ansible/hosts as it did not pass its verify_file() method
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
Loading collection community.grafana from /usr/local/lib/python3.8/dist-packages/ansible_collections/community/grafana
Loading callback plugin default of type stdout, v2.0 from /usr/local/lib/python3.8/dist-packages/ansible/plugins/callback/default.py
Skipping callback 'default', as we already have a stdout callback.
Skipping callback 'minimal', as we already have a stdout callback.
Skipping callback 'oneline', as we already have a stdout callback.

PLAYBOOK: playbook.yml ****************************************************************************************************************************************************************************************************
Positional arguments: playbook.yml
verbosity: 4
connection: smart
timeout: 10
become_method: sudo
tags: ('all',)
inventory: ('/etc/ansible/hosts',)
forks: 5
1 plays in playbook.yml

PLAY [localhost] **********************************************************************************************************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************************************************************************************************
task path: /home/k8s/grafana-dashboards/test/playbook.yml:1
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: k8s
<127.0.0.1> EXEC /bin/sh -c 'echo ~k8s && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/k8s/.ansible/tmp `"&& mkdir "` echo /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614 `" && echo ansible-tmp-1655458180.2395077-375596-199207899461614="` echo /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614 `" ) && sleep 0'
Using module file /usr/local/lib/python3.8/dist-packages/ansible/modules/setup.py
<127.0.0.1> PUT /home/k8s/.ansible/tmp/ansible-local-375591v_0x2w1v/tmp4wgq8ssy TO /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614/AnsiballZ_setup.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614/ /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614/AnsiballZ_setup.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614/AnsiballZ_setup.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /home/k8s/.ansible/tmp/ansible-tmp-1655458180.2395077-375596-199207899461614/ > /dev/null 2>&1 && sleep 0'
ok: [localhost]
META: ran handlers

TASK [Import Grafana RabbitMQ-Overview Dashboard] *************************************************************************************************************************************************************************
task path: /home/k8s/grafana-dashboards/test/playbook.yml:4
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: k8s
<127.0.0.1> EXEC /bin/sh -c 'echo ~k8s && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/k8s/.ansible/tmp `"&& mkdir "` echo /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077 `" && echo ansible-tmp-1655458181.078178-375693-36992039315077="` echo /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077 `" ) && sleep 0'
Using module file /usr/local/lib/python3.8/dist-packages/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py
<127.0.0.1> PUT /home/k8s/.ansible/tmp/ansible-local-375591v_0x2w1v/tmpd9qsj5i2 TO /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/ /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py", line 107, in <module>
    _ansiballz_main()
  File "/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', init_globals=dict(_module_fqn='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', _modlib_path=modlib_path),
  File "/usr/lib/python3.8/runpy.py", line 207, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_community.grafana.grafana_dashboard_payload_f4zkdj2y/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py", line 559, in <module>
  File "/tmp/ansible_community.grafana.grafana_dashboard_payload_f4zkdj2y/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py", line 514, in main
KeyError: 'grafana_url'
fatal: [localhost]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/k8s/.ansible/tmp/ansible-tmp-1655458181.078178-375693-36992039315077/AnsiballZ_grafana_dashboard.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', init_globals=dict(_module_fqn='ansible_collections.community.grafana.plugins.modules.grafana_dashboard', _modlib_path=modlib_path),\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_community.grafana.grafana_dashboard_payload_f4zkdj2y/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py\", line 559, in <module>\n  File \"/tmp/ansible_community.grafana.grafana_dashboard_payload_f4zkdj2y/ansible_community.grafana.grafana_dashboard_payload.zip/ansible_collections/community/grafana/plugins/modules/grafana_dashboard.py\", line 514, in main\nKeyError: 'grafana_url'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}

PLAY RECAP ****************************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Comment by rrey** [comment]
Fix was release in 1.5.0 that should be available in few minutes in Galaxy.

**Comment by itaytalmi** [comment]
Thank you @rrey, I can confirm that it works in 1.5.0 indeed.
